boards: nordic: ipc: added dcache alignement

doki-nordic opened the pull request:
The nRF54 and nRF92 chips has data cache, which means the ICMsg and ICBMsg must be configured to follow required cache alignment of the shared memory.
The `dcache-alignement` needs to be defined for that.

peknis commented on doc/services/ipc/ipc_service/backends/ipc_service_icbmsg.rst:
```diff
@@ -40,12 +40,21 @@ Configuration
 * Define MBOX devices for sending a signal that informs the other domain (or CPU) of the written data.
   Ensure that the other domain (or CPU) can receive the signal.
 
+.. warning::
```
```suggestion
.. caution::
```
Warning is too strong.

peknis commented on doc/services/ipc/ipc_service/backends/ipc_service_icbmsg.rst:
```diff
@@ -40,12 +40,21 @@ Configuration
 The backend is configured using Kconfig and devicetree.
 When configuring the backend, do the following:
 
+* If at least one of the cores uses data cache on shared memory, set ``dcache-alignment``.
```
```suggestion
* If at least one of the cores uses data cache on shared memory, set the ``dcache-alignment`` value.
```

peknis replied:
Same fixes to the other doc.

peknis commented on doc/services/ipc/ipc_service/backends/ipc_service_icbmsg.rst:
```diff
@@ -40,12 +40,21 @@ Configuration
 When configuring the backend, do the following:
 
+* If at least one of the cores uses data cache on shared memory, set ``dcache-alignment``.
+  This must be the biggest value of the invalidation or the write-back size for both sides of the communication.
```
```suggestion
  This must be the largest value of the invalidation or the write-back size for both sides of the communication.
```

doki-nordic committed:
boards: nordic: ipc: added dcache alignement

The nRF54 and nRF92 chips has data cache, which means
the ICMsg and ICBMsg must be configured to follow required
cache alignment of the shared memory.
The `dcache-alignement` needs to be defined for that.

Signed-off-by: Dominik Kilian <Dominik.Kilian@nordicsemi.no>
